common/config/MeleeCodes.c

```diff
@@ -50,23 +52,45 @@ const MeleeCodeOption cfOptionStealthUcf = {
 	g_ucf_stealth, // code
 };
 
+const MeleeCodeOption cfOptionUcf084 = {
+	4, // value
+	"UCF 0.84 RC", // name
+	g_ucf_084_size, // codeLen
+	g_ucf_084, // code
+};
+
+const MeleeCodeOption cfOptionStealthUcf084 = {
+	5, // value
+	"Stealth 0.84", // name
+	g_ucf_084_stealth_size, // codeLen
+	g_ucf_084_stealth, // code
+};
+
 const MeleeCodeOption *cfOptions[MELEE_CODES_CF_OPTION_COUNT] = {
 	&cfOptionOff,
 	&cfOptionUcf,
 	&cfOptionStealthUcf,
+	&cfOptionUcf084,
+	&cfOptionStealthUcf084,
 };
 
 static const char *cfDescription[] = {
 	"The type of controller fix to apply",
 	"",
-	"  [UCF]",
-	"    Will enable UCF 0.8. Includes",
-	"    dashback, shield drop, and",
-	"    wiggle out of tumble.",
+	"   [UCF]",
+	"     Will enable UCF 0.8. Includes",
+	"     dashback, shield drop, and",
+	"     wiggle out of tumble",
 	"",
```

Comment:
We should probably remove the extra spaces that were added here right? I'm pretty sure all the other code descriptions use two spaces to indent and now all the descriptions for controller fix use 3.

Reply:
added a comment about why the extra spacing exists